3.10/aql/operations-update.md

```diff
@@ -167,64 +219,62 @@ update non-existing documents or violating unique key constraints:
 
 ```aql
 FOR i IN 1..1000
-  UPDATE {
-    _key: CONCAT('test', i)
-  } WITH {
-    foobar: true
-  } IN users OPTIONS { ignoreErrors: true }
+  UPDATE CONCAT('test', i)
+  WITH { foobar: true } IN users
+  OPTIONS { ignoreErrors: true }
 ```
 
-An update operation will only update the attributes specified in `document` and
+An update operation only update the attributes specified in `document` and
 leave other attributes untouched. Internal attributes (such as `_id`, `_key`, `_rev`,
-`_from` and `_to`) cannot be updated and are ignored when specified in `document`.
-Updating a document will modify the document's revision number with a server-generated value.
+`_from` and `_to`) cannot be modified and are ignored when specified in `document`.
+Updating a document modifies the document's revision number with a server-generated value.
 
 ### `keepNull`
 
-When updating an attribute with a null value, ArangoDB will not remove the attribute 
-from the document but store a null value for it. To get rid of attributes in an update
-operation, set them to null and provide the `keepNull` option:
+When updating an attribute to a `null` value, ArangoDB does not remove the attribute 
+from the document but stores this `null` value. To remove attributes in an update
+operation, set them to `null` and set the `keepNull` option to `false`:
 
```

Comment:
It is not clear from the text whether in this case also attributes that have been set to `null` _previously_ are deleted.

Reply:
Added "This removes the attributes you specify but not any previously stored attributes with the `null` value"